src/oemof/tabular/facades.py

```diff
@@ -402,30 +466,27 @@ class Dispatchable(Source, Facade):
     ...         'min': 0.2})
 
     """
+    bus: Bus
 
-    def __init__(self, *args, **kwargs):
-        kwargs.update({"_facade_requires_": ["bus", "carrier", "tech"]})
-        super().__init__(*args, **kwargs)
+    carrier: str
 
-        self.profile = kwargs.get("profile", 1)
+    tech: str
 
-        self.capacity = kwargs.get("capacity")
+    profile: float = 1
 
-        self.capacity_potential = kwargs.get(
-            "capacity_potential", float("+inf")
-        )
+    capacity: float = 0
 
-        self.marginal_cost = kwargs.get("marginal_cost", 0)
+    capacity_potential: float = float("+inf")
 
-        self.capacity_cost = kwargs.get("capacity_cost")
+    marginal_cost: float = 0
```

Comment:
Here, the behavior is changed. In the first step, the behavior should remain the same.